sentry/src/main/java/io/sentry/MainEventProcessor.java

```diff
@@ -90,6 +91,13 @@ private void setDebugMeta(final @NotNull SentryEvent event) {
     }
   }
 
+  private void setModules(final @NotNull SentryEvent event) {
+    final Map<String, String> modules = event.getModules();
+    if (modules == null) {
+      event.setModules(options.getModulesLoader().getOrLoadModules());
+    }
+  }
```

Comment:
What if we append the items if there are some already?

Reply:
yeah we could do that, what would be the usecase tho? Hybrid SDKs setting their packages and we're appending to them? 

If we do that, can the MainEventProcessor run twice for the same event, so we're potentially doubling the modules list in that case?

Reply:
`modules` is part of the public API, and people can raise events with their own modules, the current behavior is to keep theirs or ours, I'd prefer to send them both (Imagine hybrid SDKs here with JS modules + Native modules, that is awesome and right now it won't be possible).
Event processors run only once per event, so I'm not aware of how this could happen.

Reply:
Cool, thanks, I'll change it then 👍